scripts/src/components/organisation-form.js

```diff
@@ -142,11 +156,12 @@ var CreateOrg = React.createClass({
    * @returns {object}
    */
   _renderReferenceLinks: function() {
-    const refLinks = [];
+    const refLinks = [],
+          redirectIdType = this.state.referenceLinks.redirect_id_type,
+          defaultRedirect = (redirectIdType) ? redirectIdType : 'unset';
```

Comment:
Does Javascript allow use of redirectIdType like this? (Since it's being initialised at the same time)